server/bundler/loader.js

```diff
@@ -84,7 +83,7 @@ function splitTemplate( path, section ) {
 		'page( ' + pathRegex + ', function( context, next ) {',
 		'	if ( _loadedSections[ ' + JSON.stringify( section.module ) + ' ] ) {',
 		'		controller.setSection( ' + JSON.stringify( section ) + ' )( context );',
-		'		layoutFocus.next();',
+		'		context.store.dispatch( { type: "LAYOUT_NEXT_FOCUS_ACTIVATE" } );',
```

Comment:
>I tried that and it broke Calypso (see a few commits back)

Looking at 05eb2dd02cacd3656cd8a8a5b3eb22d8704687f4, I think the issue is that you were trying to assign the action creator as the entire `actions` module, when it should have been:

```diff
-"\tactivateNextLayoutFocus = require( 'state/ui/layout-focus/actions' ),",
+"\tactivateNextLayoutFocus = require( 'state/ui/layout-focus/actions' ).activateNextLayoutFocus,",
```